common/djangoapps/adaptive_learning/tests/test_utils.py

```diff
@@ -0,0 +1,127 @@
+"""
+Tests for utils of adaptive_learning app.
+"""
+
+from django.test import TestCase
+from mock import Mock, patch
+
+from adaptive_learning.tests.base import AdaptiveLearningTestMixin
+from adaptive_learning.utils import get_pending_reviews, make_revisions
+
+
+class AdaptiveLearningUtilsTest(TestCase, AdaptiveLearningTestMixin):
```

Comment:
@pomegranited Huh. For some reason I thought it was the other way around :) Made that change.